pontoon/sync/vcs/models.py

```diff
@@ -114,6 +158,10 @@ def __init__(
 
         self.configuration = None
         if db_project.configuration_file:
+            # Permalink is required to download project config files.
+            if db_project.source_repository.permalink_prefix:
```

Comment:
`if not`?

Reply:
`if not` is on my local machine already :D Sorry for pushing partial changes. I'll ping you when it's ready for the next stage of review.